1/src/Shape.spec.ts

```diff
@@ -0,0 +1,32 @@
+// @ts-nocheck
```

Comment:
One of the requirements of the task is to remove "@ts-nocheck"s and your code should work without any issue after that.

Reply:
ok, remove it and run tests 
![image](https://github.com/varchenkojulia1/software-designs-and-algorithms/assets/72207362/4d00395d-97fa-4abc-855a-859fbd039b31)